go.mod

```diff
@@ -1,4 +1,4 @@
-module nrversions
+module newrelic-versions
```

Comment:
One of the silly decisions of Go is that this is an invalid module name. https://go.dev/blog/package-names

We should keep it as `nrversions`. My #5 PR currently builds and issues releases with `nrversions`.